### PR TITLE
JBPM-5686 - Kie scanner scanning interval

### PR DESCRIPTION
**Pull request by mswiderski**

### CODE REVIEW

**Comment by manstis** [comment]
@karreiro Would you mind taking a look too please?

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/config/rules/ContainerRulesConfigPresenter.java [hunk 1]
```diff
@@ -146,7 +161,41 @@ public boolean error( final Object o,
...
+            return interval;
+        } else if (timeUnit.equalsIgnoreCase(S)) {
+            return interval / SECOND_IN_MILLIS;
+        } else if (timeUnit.equalsIgnoreCase(M)) {
+            return interval / MINUTE_IN_MILLIS;
+        } else if (timeUnit.equalsIgnoreCase(H)) {
+            return interval / HOUR_IN_MILLIS;
+        } else if (timeUnit.equalsIgnoreCase(D)) {
+            return interval / DAY_IN_MILLIS;
+        }
```
Comment (karreiro) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/config/rules/ContainerRulesConfigPresenter.java]: I think that this method is not covered by tests.

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/config/rules/ContainerRulesConfigView.java [hunk 2]
```diff
@@ -134,6 +140,32 @@ public void onChange( ChangeEvent event ) {
...
+
+        Option h = new Option();
+        h.setText(translationService.format( Constants.ContainerRulesConfigView_Hours ));
+        h.setValue(ContainerRulesConfigPresenter.H);
+        intervalTimeUnit.add(h);
+
+        Option d = new Option();
+        d.setText(translationService.format( Constants.ContainerRulesConfigView_Days ));
+        d.setValue(ContainerRulesConfigPresenter.D);
+        intervalTimeUnit.add(d);
```
Comment (karreiro) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/config/rules/ContainerRulesConfigView.java]: I've noticed that you're using GWT Bootstrap 3 classes. Is there any reason for not to use ERRAI UI here?

**Comment by manstis** [comment]
@mswiderski @karreiro What's the status of this please?

**Comment by mswiderski** [comment]
@manstis missed this review comments somehow, will work on it soon.

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/config/rules/ContainerRulesConfigView.java (hunk 2 above)
Comment (mswiderski) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/config/rules/ContainerRulesConfigView.java]: @karreiro that was recommended by both @manstis and @Rikkola for static data

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/config/rules/ContainerRulesConfigPresenter.java (hunk 1 above)
Comment (mswiderski) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/config/rules/ContainerRulesConfigPresenter.java]: will add tests for it

**Comment by mswiderski** [comment]
@manstis @karreiro changes applied

**Comment by mswiderski** [comment]
@karreiro no need to be sorry about it. Does it look better now?

**Pull request merged**